### PR TITLE
Add random suffix to trial runtimes

### PR DESCRIPTION
**Pull request by kjaksik**
**Description**

Changes proposed in this pull request:

- Add random suffix to trial runtimes

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/process/input/input.go [hunk 1]
```diff
@@ -268,6 +278,15 @@ func (r *RuntimeInput) applyGlobalOverridesForUpgradeRuntime() error {
 	return nil
 }
 
+func (r *RuntimeInput) addRandomStringToTrialRuntimeName() error {
+	rand.Seed(time.Now().UnixNano())
+	if r.provisioningParameters.PlanID == broker.TrialPlanID {
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/process/input/input.go:283]: there is an IsTrial func in the broker package

Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/process/input/input.go:283]: Why the random part is used only for trial? are there any business reasons?

Comment (kjaksik) [reply components/kyma-environment-broker/internal/process/input/input.go]: User cannot specify the name when provisioning a trial skr. A random name comes from CP and there were occurences when the name was repeated. We want to make sure that the name is random and it won't cause any problems in the Director DB

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**